api/src/main/java/org/pmiops/workbench/billing/BillingGarbageCollectionService.java

```diff
@@ -58,6 +58,22 @@ public BillingGarbageCollectionService(
     this.credentialsLoadingCache = initializeCredentialsLoadingCache();
   }
 
+  public void deletedWorkspaceGarbageCollection() {
```

Comment:
Changed to public because it is now called by a Controller outside of this package